CityOfDetroit/bloom
# 1252/mobile fixes

## Issue #1252 

- Closes #1252 
- Addresses #issue

## Description

This PR addresses two unrelated fixes -  1) a RTL fix on Safari in which the the layout wasn't switching back to LTR after going from Arabic back to non-RTL language, 2) A spacing issue between the $ per month and % income section on the listings tables,

## How Can This Be Tested/Reviewed?

Entry Point: http://localhost:3000/listings | https://detroit-public-dev.netlify.app/listings

Issue 1:
1. Visit entry point on Safari
2. Switch language to Arabic
3. Switch language back to English (or other non-RTL language)
4. Notice that page layout is properly displaying left-to-right

Issue 2:
1. On same page, minimize window to mobile width
2. Notice that there is space between the after the comma between $ per month and % income:
![Screen Shot 2022-05-10 at 11 53 52 AM](https://user-images.githubusercontent.com/60076088/167688093-96cbc637-3e88-4e7c-bf69-2ee9b0ef7293.png)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
